sky/data/storage_utils.py

```diff
@@ -120,91 +123,99 @@ def get_excluded_files_from_gitignore(src_dir_path: str) -> List[str]:
     """
     expand_src_dir_path = os.path.expanduser(src_dir_path)
 
-    git_exclude_path = os.path.join(expand_src_dir_path, '.git/info/exclude')
-    gitignore_path = os.path.join(expand_src_dir_path,
-                                  constants.GIT_IGNORE_FILE)
+    # We will use `git ls-files` to list files that we should ignore, but
```

Comment:
Can we add the behavior of the `gitignore` to the docstr as well for better reference?